### PR TITLE
Jason/327 compatibility tag filter

### PR DESCRIPTION
**Pull request by duidae**
在舊的`標籤`欄位選項過濾掉新增的子分類，@Aylie-Chou 再拜託幫忙確認一下新增的子分類是否正確，3Q

RelationshipFieldForTag/RelationshipFieldForCategory差異可以diff:
vimdiff fields/types/relationship/RelationshipFieldForTag.js fields/types/relationship/RelationshipField.js
vimdiff fields/types/relationship/RelationshipFieldForCategory.js fields/types/relationship/RelationshipField.js

### CODE REVIEW

File: fields/types/relationship/RelationshipFieldForTag.js [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+      url: Keystone.adminPath + '/api/' + this.props.refList.path + '?basic&search=' + input + '&' + filters,
+      responseType: 'json',
+    }, (err, resp, data) => {
+      if (err) {
+        console.error('Error loading items:', err);
+        return callback(null, []);
+      }
+      data.results.forEach(this.cacheItem);
+      callback(null, {
+        options: data.results.filter(subcategory => !NEW_SUBCATEGORY_IDS.includes(subcategory.id)),
```
Comment (Aylie-Chou) [inline fields/types/relationship/RelationshipFieldForTag.js:149]: The feature that `tag` options would exclude those tag marked as `subcategory` will need to be obtain in the future (even afrer new info structure release).
For the future need, would it be better if we change the filter to filter out tags that has `category` field ?

Comment (duidae) [reply fields/types/relationship/RelationshipFieldForTag.js:149]: unused fields (such as `category` field) is automatically filtered out by keystone, so we dont have such fields to obtain here

Screen Shot 2022-10-03 at 2 40 12 PM

Comment (Aylie-Chou) [reply fields/types/relationship/RelationshipFieldForTag.js:149]: Got it !
I think this problem could be wait until developing category set edition in the future.

**Comment by Aylie-Chou** [comment]
LGTM!

**Pull request merged**